wasi-sdk-p2.cmake

```diff
@@ -0,0 +1,40 @@
+# Cmake toolchain description file for the Makefile
```

Comment:
the file needs to be moved into the `docker` folder

Reply:
Ah, thanks for pointing that out.  TIL there are two versions of each of the wasi-sdk*.cmake files.